Sources/MapboxDirections/RoadClasses.swift

```diff
@@ -43,6 +43,21 @@ public struct RoadClasses: OptionSet, CustomStringConvertible {
      */
     public static let tunnel = RoadClasses(rawValue: 1 << 5)
     
+    /**
+     The road segment is a HOV-2 road that requires a minimum of two vehicle occupants.
+    */
+    public static let hov2 = RoadClasses(rawValue: 1 << 6)
+    
+    /**
+     The road segment is a HOV-3 road that requires a minumum of three vehicle occupants.
+    */
+    public static let hov3 = RoadClasses(rawValue: 1 << 7)
+    
+    /**
+     The road segment is a HOT road that is tolled if the user's vehicle does not meet the minimum occupant requirement.
+    */
+    public static let hot = RoadClasses(rawValue: 1 << 8)
```

Comment:
The term “HOT” is traffic engineering/management jargon that isn’t widely known among software developers in the U.S., let alone other countries. `hot` is also prone to be misinterpreted and look out of place as the adjective. Spell out `highOccupancyToll` and link “high-occupancy toll road” to [this Wikipedia article](https://en.wikipedia.org/wiki/High-occupancy_toll_lane). The article is about HOT lanes and express toll lanes, so we should clarify that the option is about HOT-only roads, not mere HOT lanes.

Reply:
Oh that makes sense. I will fix this for hot and hov.